pandas/tests/reshape/concat/test_append.py

```diff
@@ -350,13 +350,14 @@ def test_append_empty_frame_to_series_with_dateutil_tz(self):
         )
         expected["c"] = expected["c"].astype(object)
         expected["d"] = expected["d"].astype(object)
-
-        result_b = result_a.append(s, ignore_index=True)
+        result_b = result_a.append(ser, ignore_index=True)
         tm.assert_frame_equal(result_b, expected)
 
         # column order is different
         expected = expected[["c", "d", "date", "a", "b"]]
-        result = df.append([s, s], ignore_index=True)
+        dtype = Series([date]).dtype
+        expected["date"] = expected["date"].astype(dtype)
```

Comment:
Is this still needed? (might be a left-over from astyping it to object before)

Reply:
you're right, updated